modules/drivers/snowflake/test/metabase/driver/snowflake_test.clj

```diff
@@ -201,8 +201,8 @@
                   ["2014-08-02T00:00:00Z" "2014-08-02T09:30:00Z"]]
                  (run-query))))
         (testing "with report timezone set"
-          (is (= [["2014-08-02T00:00:00-07:00" "2014-08-02T05:30:00-07:00"]
-                  ["2014-08-02T00:00:00-07:00" "2014-08-02T02:30:00-07:00"]]
+          (is (= [["2014-08-02T00:00:00-07:00" "2014-08-02T12:30:00-07:00"]
```

Comment:
the previous tests were wrong because we inserted the wrong type for `last_login` column

in our dataset, we define `last_login` as `:type/Datetime` so it should have type `timestamp without time zone`.

But for some reason, we previously insert it as `timestamp with local time zone` so that's why this test had value changed from `"2014-08-02T12:30:00Z"` to `"2014-08-02T05:30:00-07:00"` if you change the report-tz

then why it changed?
yea because I fixed it... [here](https://github.com/metabase/metabase/pull/26633/files#r1030090226)

it was not failing for my PR because we use a shared db to test snowflake and we don't reinsert the dataset that already got inserted. 

so this test only failing now because I removed then re-insert the data with the fixed above.